### PR TITLE
update geoip database source for all region

### PR DESCRIPTION
**Pull request by nick322**
because config update_url :https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz 
no recognizable all city

### CODE REVIEW

**Comment by nick322** [comment]
@Torann Approve please

File: composer.json [hunk 1]
```diff
@@ -1,65 +1,65 @@
-{
```
Comment (Torann) [inline composer.json]: What's going on with this diff?

File: config/geoip.php [hunk 2]
```diff
@@ -54,7 +54,7 @@
         'maxmind_database' => [
             'class' => \Torann\GeoIP\Services\MaxMindDatabase::class,
             'database_path' => storage_path('app/geoip.mmdb'),
-            'update_url' => 'https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz',
```
Comment (Torann) [inline config/geoip.php]: Why did you change this? https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz is still a valid download

File: composer.json (hunk 1 above)
Comment (nick322) [reply composer.json]: Because there was a commit (https://github.com/Torann/laravel-geoip/pull/143/commits/dbf80a28593a569f06c49f7be3bc229591dd86e0), it was a failure, so I changed composer.json to do a travis-ci test, after which he prompted I passed, I corrected it back to the original version (https://github.com/Torann/laravel-geoip/pull/143/commits/a2ccbc530f0e64cf279e1f1389017300e7935d29),
I don't know if this affects you.
I feel sorry if I have it.

File: config/geoip.php (hunk 2 above)
Comment (nick322) [reply config/geoip.php]: As provided by the official (https://dev.maxmind.com/geoip/geoip2/geolite2/)
The link is (https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.tar.gz) tar.gz compression, compressed file is smaller than the original (https://geolite.maxmind.com/ Download/geoip/database/GeoLite2-City.mmdb.gz)

Comment (nick322) [reply config/geoip.php]: There is currently no mention that it will be removed, but the link is not visible on the official website. Understood!  I will close the issue :)
